### PR TITLE
ticdc: add two troubleshoot QAs

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add two troubleshoot QAs for TiCDC.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5047 https://github.com/pingcap/docs-cn/pull/5083
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -121,6 +121,14 @@ The TiCDC server chooses its time zone in the following priority:
 2. When `--tz` is not available, TiCDC tries to read the time zone set by the `TZ` environment variable.
 3. When the `TZ` environment variable is not available, TiCDC uses the default time zone of the machine.
 
+## What is the default behavior of TiCDC if I create a replication task without specifying the configuration file in `--config`?
+
+If you use the `cdc cli changefeed create` command without specifying the `-config` parameter, TiCDC creates the replication task according to the following steps:
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:126]:
```suggestion
If you use the `cdc cli changefeed create` command without specifying the `-config` parameter, TiCDC creates the replication task in the following default behaviors:
```

**Review by leoppro — APPROVED** [review]
LGTM

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -121,6 +121,14 @@ The TiCDC server chooses its time zone in the following priority:
...
+
+* Replicate all tables except system tables
+* Disable the Old Value feature
+* Skip replicating tables that do not contain [valid indexes](/ticdc/ticdc-overview.md#restrictions)
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:130]:
```suggestion
* Replicates all tables except system tables
* Disables the Old Value feature
* Skips replicating tables that do not contain [valid indexes](/ticdc/ticdc-overview.md#restrictions)
```

File: ticdc/troubleshoot-ticdc.md [hunk 3]
```diff
@@ -245,3 +253,13 @@ In TiCDC Open Protocol, the type code `6` represents `null`.
 | Null | 6 | `{"t":6,"v":null}` | |
 
 For more information, refer to [TiCDC Open Protocol column type code](/ticdc/ticdc-open-protocol.md#column-type-code).
+
+## How can I tell if a Row Changed Event of TiCDC Open Protocol is an `INSERT` event or an `UPDATE` event?
+
+If the the Old Value feature is not enabled, you cannot tell whether a Row Changed Event of TiCDC Open Protocol is an `INSERT` event or an `UPDATE` event. If the feature is enabled, you can determine the event type by the fields it contains:
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:259]:
```suggestion
If the Old Value feature is not enabled, you cannot tell whether a Row Changed Event of TiCDC Open Protocol is an `INSERT` event or an `UPDATE` event. If the feature is enabled, you can determine the event type by the fields it contains:
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM
@leoppro PTAL

**Comment by leoppro** [comment]
LGTM again

**Pull request merged**